2019q1/bcm58712.md

```diff
@@ -0,0 +1,24 @@
+## Broadcom ARM64 SoC support ##
+
+Contact: Michal Stanek, <mst@semihalf.com>, Marcin Wojtas, <mw@semihalf.com>
+
+Semihalf team work continuation on a FreeBSD support for Broadcom BCM5871X SoC series.
+
+Link: [BCM58712 Product Brief](https://www.broadcom.com/products/embedded-and-networking-processors/communications/bcm58712/)
+
+BCM5871X are quad-core 64-bit ARMv8 Cortex-A57 communication processors targeted for
+networking applications such as 10G routers, gateways, control plane processing and NAS.
+
+* Completed since the last update:
+   * iProc PCIe root complex (internal and external buses)
+   * OTP (One Time Programmable memory) driver
+
+* In progress:
+  * BNXT Ethernet support
+  * Crypto engine acceleration for IPsec offloading.
+
+* Todo:
+  * Upstreaming - this work is expected to be submitted to HEAD in the second half of 2019.
+
+Sponsored by: Juniper
```

Comment:
Should that be `Juniper Networks` or `Juniper Networks, Inc.` perhaps?

Reply:
Done